workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap

```diff
@@ -1,26 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
-.emotion-2 {
```

Comment:
Do you know why the dependency update changes the snapshots?

Reply:
`frontend-react-developer` is updated and forces `emotion` to upgrade from 9.2.12 to 10.0.0. I think this is the reason for the different snapshots.

Reply:
👍